driver/pelican/demandResponse.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/immesys/spawnpoint/spawnable"
+	bw2 "gopkg.in/immesys/bw2bind.v5"
+)
+
+type ADREventWrapperAPI struct {
+	XMLName xml.Name    `xml:"result"`
```

Comment:
Hm I don't think I've seen `XMLName` used in the rest of the Pelican code. Why do we need it here? If we don't, we probably want to keep things consistent.